camel/societies/role_playing.py

```diff
@@ -291,6 +291,8 @@ def init_agents(
                 function_list=self.assistant_functions,
                 function_call="auto",
             )
+        elif assistant_agent_kwargs:
+            assistant_config = assistant_agent_kwargs.pop("model_config", None)
```

Comment:
Sorry that this was implemented in previous PR about function calling. This was because I thought the actual execution should only be done by the assistant while the user is focused on giving instructions without the need to do function calling. Does this make sense or do you have any suggestions on this?